encrypted-media-respec.html

```diff
@@ -3108,19 +3108,63 @@
+          Persistent Data includes all data stored by the CDM, or by the User Agent on behalf of the CDM, that exists after the destruction of the <a>MediaKeys</a> object. Specifically, it includes any identifiers, licenses, keys, key IDs, key usage information or records of license destruction stored by the CDM or by the User Agent on behalf of the CDM.
         </p>
+        <section id="use-origin-specific-key-system-storage">
+          <h3>Use origin-specific and browsing profile-specific Key System storage</h3>
+          <p>Persistent Data that might impact messages or behavior in an application- or license server-visible way MUST be stored in an <a def-id="origin"></a>-specific and <a def-id="browsing-profile"></a>-specific way and MUST NOT leak to or from private browsing sessions.
+                Specifically but not exhaustively, session data, licenses, keys and per-origin identifiers MUST be stored per-<a def-id="origin"></a> and per-<a def-id="browsing-profile"></a>.
+          </p>
+          <p>
+                See <a def-id="session-storage"></a>.
+          </p>
+        </section>
         <section id="allow-persistent-data-cleared">
           <h3>Allow Persistent Data to Be Cleared</h3>
           <p>
-            Implementations that use persistent data MUST allow the user to clear that data such that it is no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
-          </p>
-          <p id="allow-persistent-data-cleared-with-cookies">
-            Implementations SHOULD allow users to clear persistent data along with cookies [[!COOKIES]] and other site data.
-            It is RECOMMENDED that users be offered the option to clear persistent data as part of user agent features to clear browsing history.
+            Implementations that use Persistent Data MUST allow the user to clear that data such that it is no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
           </p>
           <p>
-            It is RECOMMENDED that users be able to request that persistent data be forgotten on a per-origin basis, particularly as part of a "Forget about this site" feature that forgets cookies [[!COOKIES]], databases, etc. associated with a particular site in an operation that is sufficiently atomic to prevent "cookie resurrection" type of recorrelation of a new identifier with the old by relying on another type of locally stored data that did not get cleared at the same time.
+            User Agents SHOULD:
+          </p>
+          <ul>
+            <li>
+              <p id="allow-persistent-data-cleared-with-cookies">
+                allow users to clear Persistent Data along with cookies [[!COOKIES]] and other site data.
```

Comment:
I believe we have capitalized them in such cases. See "In summary:" and "Among other things, this means that:"